default-templates/server/index.html

```diff
@@ -30,6 +26,41 @@ <h1>Welcome to the Solid Prototype</h1>
 
     If you have not already done so, please create an account.
   </p>
+
+  <p class="lead hidden" id="loggedIn">
+    You are logged in as
+    <a href="#" id="profileLink"></a>.
+  </p>
 </div>
+<script src="/common/js/solid-auth-client.bundle.js"></script>
+<script>
+  const elements = {};
```

Comment:
We should be vary of using syntax such as const and let in the client, as these are not supported by all browsers: https://caniuse.com/#search=const

Which begs the question: Which browsers do we officially support?

Reply:
This PR copies over functionality from `default-views/auth/login-required.hbs`, which uses the same syntax. So I agree with the comment, but this decision has already been made elswhere, i.e., this PR is not breaking things that were working before.

Reply:
Ok, do we know where that decision is documented?